### PR TITLE
[15.0][MIG] partner_email_check

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
@ForgeFlow
Migration to 15.0

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by quannguyen1560 — COMMENTED** [review]
It would be nice if you can check the warning and fix it.

**Comment by Rikylora30** [comment]
ready? Why don't they close it?

**Review by leemannd — APPROVED** [review]

File: partner_email_check/tests/test_partner_email_check.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from unittest.mock import patch
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import SavepointCase
+from odoo.tools.misc import mute_logger
+
+
+class TestPartnerEmailCheck(SavepointCase):
```
Comment (rousseldenis) [inline partner_email_check/tests/test_partner_email_check.py]: @JuanyDForgeflow @LoisRForgeFlow Could you change this to TransactionCase ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Change test

File: partner_email_check/tests/test_partner_email_check.py (hunk 1 above)
Comment (yankinmax) [reply partner_email_check/tests/test_partner_email_check.py]: @JuanyDForgeflow @LoisRForgeFlow can you pls either update PR with TransactionCase or give access to do it?

Comment (JuanyDForgeflow) [reply partner_email_check/tests/test_partner_email_check.py]: @rousseldenis @yankinmax I just changed it

**Comment by JuanyDForgeflow** [comment]
@rousseldenis 

> Change test

It's ok now

**Comment by yankinmax** [comment]
@rousseldenis ping

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**